doc/HOW_YOU_CAN_HELP.md

```diff
@@ -0,0 +1,21 @@
+# How You Can Help
```

Comment:
This kind of thing goes out of date FAST.
It's a lot more scalable to add a search link for e.g. a particular issue tag, like https://github.com/CleverRaven/Cataclysm-DDA/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22 and then focus some effort on maintaining that tagging.

Reply:
I see your point. If it's in a self-contained issue, then the details can be a lot more expansive, and people can comment on it and reference it in commits.

I think instead, we should have a little guide on how to search for these tags, as well as a list of the various tags.

Reply:
What are your thoughts on the new one?

Reply:
Wonderful :+1: